doc/developers/contributing.rst

```diff
@@ -302,18 +302,31 @@ following rules before submitting:
    or link to a `gist <https://gist.github.com>`_. If an exception is raised,
    please provide the traceback.
 
-.. _easy_issues:
+.. _new_contributors:
 
-Easy Issues
------------
+Issues for New Contributors
+----------------
```

Comment:
The dotted line is too short compared to the title. This will not be rendered properly.